app/controllers/miq_policy_controller/miq_actions.rb

```diff
@@ -432,8 +432,10 @@ def action_get_info(action)
 
     if %w(inherit_parent_tags remove_tags).include?(@action.action_type)
       tag = Tag.find_by_classification_name(@action.options[:cats])
-      cats = Classification.where(:tag_id => tag.id).pluck(:description)
-      @cats = cats.sort_by(&:downcase).join(" | ")
+      if tag.present?
```

Comment:
What would the scenarios be for this being present/not present? Could we add a test for both?